source/table.cpp

```diff
@@ -41,7 +41,15 @@ spv_context spvContextCreate(spv_target_env env) {
   spvOperandTableGet(&operand_table, env);
   spvExtInstTableGet(&ext_inst_table, env);
 
-  return new spv_context_t{env, opcode_table, operand_table, ext_inst_table};
+  return new spv_context_t{
+      env, opcode_table, operand_table, ext_inst_table,
+      nullptr /* an empty MessageConsumer */
```

Comment:
That's reasonable.

It's just conceptually more clear to use a null std::function here to mean no consumer at all. But actually doesn't bring any benefits other than that.

I'll change this to a default consumer just ignoring messages.